packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -13,6 +13,8 @@ import '../base/common.dart';
 import '../base/file_system.dart';
 import '../base/process.dart';
 import '../build_info.dart';
+import '../build_runner/devfs_web.dart';
```

Comment:
We cant let this implementation leak into the rest of the tool. There is an interface that flutter run uses, this needs to go through the same indirection

Reply:
Good to know. Have this fixed by creating a getter on ResidentRunner.